distribution/lib/Standard/Table/0.0.0-dev/src/Column.enso

```diff
@@ -2971,3 +2979,13 @@ apply_unary_map column:Column new_name:Text function expected_result_type:Value_
 Many_Files_List.from (that : Column) =
     Read_Many_Helpers.ensure_column_type_valid_to_be_files_list that <|
         Many_Files_List.Value that that.to_vector
+
+## PRIVATE
+   Temporary function to determine if to use the Comparators statics or embedded operations.
+call_comparator left other new_name java_static enso_action =
```

Comment:
```suggestion
call_comparator left other new_name java_static ~enso_action =
```
probably want to pass the fallback action as lazy as it can be costly to execute. We don't want to run the whole comparison logic twice 😅